src/main/resources/properties/configuration.properties

```diff
@@ -1,3 +1,4 @@
 configuration.profiles=web/desktop/chrome
 configuration.suites=web_app
 configuration.environments=
```

Comment:
As soon as you add environment.properties file (see recommendations in another comment) , add the link to that folder here.